connectors.go

```diff
@@ -4,30 +4,36 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"io"
 
 	"github.com/amp-labs/connectors/common"
 	"github.com/amp-labs/connectors/hubspot"
 	"github.com/amp-labs/connectors/providers"
+	"github.com/amp-labs/connectors/proxy"
```

Comment:
Seems like this is going to cause an import cycle, you might want to move some files around to avoid this.